README.md

```diff
@@ -4,52 +4,55 @@ The first ever Bitcoin Vault wallet.
 
 [![Playstore](https://bluewallet.io/img/play-store-badge.svg)](https://play.google.com/store/apps/details?id=io.goldwallet.wallet)
 
-
 ## BUILD & RUN IT
 
-* In your console:
+- In your console:
 
 ```
 git clone https://github.com/bitcoinvault/GoldWallet.git
 cd GoldWallet
-npm install
+yarn install
 ```
 
-* Add files for firebase:
+- Add files for firebase:
 
 For iOS:
 GoogleService-Info.plist into directory /ios/
 
 For android:
-google-services.json into directory /android/app/
+google-services.json into directory /config/prod/
+
+- Create .env file:
+
+Create .env file in /config/prod/ directory. It will be copied to root directory while building. Check the .env.example file for a template
 
-* Create .env file:
+- Create env specific files in config direcotry
 
-Create .env file in root directory. Check the .env.example file for a template
+in config/prod/ directory add app.json and MainActivity.java - you will find them already in the project files.
 
-* To run on Android:
+- To run on Android:
 
 ```
-npm start android
+yarn start android:prod
 ```
 
-* To run on iOS:
+if you want to have beta version with different name and id - redo previous steps for config/beta
```

Comment:
I think it is not obvious what you gotta do here. I would add something like

To run on Android beta:
Put file google-services.json into directory /config/beta/

> yarn start android:beta